lib/src/signals/logic_array.dart

```diff
@@ -127,4 +127,14 @@ class LogicArray extends LogicStructure {
     return LogicArray(dimensions, elementWidth,
         numUnpackedDimensions: numUnpackedDimensions, name: name);
   }
+
+  /// add doc later
+  void assignSubset(List<Logic> updatedSubset, int start) {
```

Comment:
did you mean to include this?  I think this was supposed to be in another PR for the other issue?